src/picongpu/include/simulation_defines/param/gridConfig.param

```diff
@@ -30,15 +30,30 @@ namespace picongpu
         /** Duration of one timestep
          *  unit: seconds */
         BOOST_CONSTEXPR_OR_CONST float_64 DELTA_T_SI = 0.8e-16;
+
         /** equals X
          *  unit: meter */
         BOOST_CONSTEXPR_OR_CONST float_64 CELL_WIDTH_SI = 0.1772e-6;
-        /** equals Y - the propagation direction
+        /** equals Y - the laser & moving window propagation direction
```

Comment:
put in this general file to point users to what the currently chosen propagation axis for settings in `laserConfig.param` is.